common/history/states/southeast_asian_states.txt

```diff
@@ -66,6 +66,7 @@ STATES = {
 		}
 		
 		add_claim = c:WPA add_homeland = west_papuan
+		add_homeland = melanesian	
```

Comment:
melanesian is to be consolidated into a national identity for each country for game performance reasons so there should be no additional homeland.